src/parser/scanner.lex

```diff
@@ -485,16 +491,7 @@ FORMAT                      ([Ff][Oo][Rr][Mm][Aa][Tt])
                                 }
                                 return TokenType::INTEGER;
                             }
-{DEC}+\.{DEC}*              {
-                                try {
-                                    folly::StringPiece text(yytext, yyleng);
-                                    yylval->doubleval = folly::to<double>(text);
-                                } catch (...) {
-                                    throw GraphParser::syntax_error(*yylloc, "Out of range:");
-                                }
-                                return TokenType::DOUBLE;
-                            }
-{DEC}*\.{DEC}+              {
+{DEC}+\.{DEC}+              {
```

Comment:
Why remove this?

Reply:
It conflicts with a new added syntax `1..2`.
But I think we could do some tricks to deal with this, maybe later on.